birdvoxpaint/display.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+
+
+def specshow(S, sr, period=60, offset=0,
+             segment_duration=10, date_offset=None,
+             fmin=0, fmax=None, y_axis='mel', sp_height=None):
+    '''Plot a false color spectrogram (or any long spectrogram for that matter).
```

Comment:
this function does not display a false-color spectrogram. it relies on a magma colormap. false color is something else entirely

Reply:
It only uses cmap if it's a single channel. It's the same thing that `plt.imshow` does. 

The only difference is that `imshow` makes you squeeze off the single channel dimension, but I figured if someone put in a single index, they'd prefer a cmap to just black/white

See the comment under cmap here: https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.imshow.html

Reply:
I see.